src/test/simple_spin.cc

```diff
@@ -46,7 +46,8 @@ TEST(SimpleSpin, Test1)
   // Should also work with pass-by-reference:
   // (Note that we don't care about cross-threading here as-such.)
   counter = 0;
-  ASSERT_EQ(counter, 0);
+  uint32_t urhs = 0;
+  ASSERT_EQ(counter, urhs);
```

Comment:
>no need to define another variable just for the comparing.

True, it's not necessary, but it avoids an ugly cast and I don't think it's a bad approach?

>and the first parameter is the expected value, the second is the actual value.
>also, what is the point of this assertion? 0 is just assigned to counter.

That code was approved in #14370 so I guess that's a separate issue?